keras/datasets/imdb.py

```diff
@@ -100,7 +100,7 @@ def load_data(path='imdb.npz', num_words=None, skip_top=0,
         for x in xs:
             nx = []
             for w in x:
-                if w >= num_words or w < skip_top:
+                if skip_top <= w < num_words:
```

Comment:
I think yes, it's safe. And if someone asks to skip more than `num_words`, then returned empty array is an adequate answer.